packages/cli/src/webhooks/webhook-helpers.ts

```diff
@@ -963,3 +851,37 @@ async function parseRequestBody(
 		}
 	}
 }
+
+/**
+ * Evaluates the `responseHeaders` parameter of a webhook node
+ */
+function evaluateResponseHeaders(
+	webhookData: IWebhookData,
+	workflow: Workflow,
+	workflowStartNode: INode,
+	executionMode: WorkflowExecuteMode,
+	additionalKeys: IWorkflowDataProxyAdditionalKeys,
```

Comment:
Would it help reduce all these long argument lists if we create an instance of a class that holds all values that won't change as we handle a webhook?

Reply:
I had a similar idea myself. Extracted them into a context object. Makes this much cleaner 🙌